stograde/student/pull.py

```diff
@@ -1,10 +1,14 @@
 import logging
 
 from ..common import chdir, run
+from ..common.run_status import RunStatus
 
 
-def pull(student: str, no_repo_update: bool = False):
+def pull(student: str):
     logging.debug("Pulling {}'s repository".format(student))
     with chdir(student):
-        if not no_repo_update:
-            run(['git', 'pull', '--quiet', 'origin', 'master'])
+        status, output, _ = run(['git', 'pull', '--quiet', 'origin', 'master'])
+
+    if status is RunStatus.CALLED_PROCESS_ERROR and 'not a git repository' in output:
+        print('Student directory {} not a git repository\n'
+              'Try running stograde repo reclone'.format(student))
```

Comment:
```suggestion
              'Try running "stograde repo reclone"'.format(student))
```

double-quotes for easy scanning of where the command to copy/paste is?